test/e2e/swaps/shared.js

```diff
@@ -157,7 +157,7 @@ const checkActivityTransaction = async (driver, options) => {
 
 const checkNotification = async (driver, options) => {
   const boxTitle = await driver.findElement(
-    '[data-testid="mm-banner-base-title"]',
+    '[data-testid="swap-token-verification-banner-title"]',
```

Comment:
Removing the static data-testid that was added to the component in favor of using `descriptionProps` and making the test id more specific"